packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -2040,6 +2042,24 @@ class ApiGateway {
   protected checkAuthSystemMiddleware: RequestHandler = async (req, res, next) => {
     await this.checkAuthWrapper(this.checkAuthSystemFn, req, res, next);
   };
+  
+  protected checkSqlRunnerScope: RequestHandler = async (req: Request, res: Response, next: NextFunction) => {
+    await this.checkAuthWrapper(
```

Comment:
Could you please clarify the behavior for non-cloud deployments? As far as I see, this makes it impossible to use the endpoint outside the cloud in prod mode. @paveltiunov, @bsod90 is it by the design?